site/topics/memory/memory.rst

```diff
@@ -2,16 +2,86 @@
 Memory
 ******
 
-* Storing data for later use is an important function of a typical digital computer
+* Until now, all circuits discussed made no use of stored information
+* However, storing data for later use is an important function of a typical digital computer
 * Memory can be used to store
 
     * Data, such as the value of a variable
     * Instructions, such as a program
 
 
 
-Bits and Bytes
-==============
+Bits, Bytes, and Data
+=====================
+
+* Before getting into the details of memory, it's important to be clear on what data and information is
+
+    * Data is some collection of raw values/symbols that have no intrinsic meaning
+
+        * For example, the number :math:`3` is data
+
+
+    * Information is data that has meaning within some context
+
+        * :math:`3` is information on a quantity in the context of the question of how many dogs are in a car
```

Comment:
I'd try to use less common words here. Outside of these definitions, data and information are used interchangably (erroneously) and it's bound to be a little confusing. Either that or really hammer home the difference, like [_easy_ vs _simple_](https://github.com/matthiasn/talk-transcripts/blob/master/Hickey_Rich/SimpleMadeEasy.md?plain=1#L23-L59) which took Rich 3 slides and nearly 10 minutes.

The students have some coding experience, so an example with some code may help, eg

```scala
// |--------| meaning (and partial context)
   val urMom = false
            // |----| data
// |----------------| information
```

Reply:
I'll think on this one on how best to approach it.

> data and information are used interchangably (erroneously) and it's bound to be a little confusing

Hell, I was working on the next section where I say that RAM in our typical computers hold data and instructions even though instructions are data.

Reply:
At the end of the day, the point of data vs information really is the difference between encoding vs meaning.